### PR TITLE
A lot of changes

### PR DESCRIPTION
**Pull request by mfatir-y**
Updated pom file, added a main class for the program to run. I also added some styling to the upload page.

### CODE REVIEW

File: src/main/java/com/intelliChef/view/RecipeUploadView.java [hunk 1]
```diff
@@ -13,23 +13,22 @@
 import java.util.List;
 
 public class RecipeUploadView extends JFrame {
-    private final JFrame frame;
     private final List<Ingredient> ingredientList = new ArrayList<>();
```
Comment (EnzaDinza) [inline src/main/java/com/intelliChef/view/RecipeUploadView.java:15]: Make sure your view doesn't access entity directly to follow CA

**Review by EnzaDinza — APPROVED** [review]
Looks great! You might want to review some parts of your code to ensure they align with Clean Architecture principles.

**Comment by narenrameshb** [comment]
The changes look good and adhere to our discussions of steps forward in group meetings. As Enza mentioned in her review, some CA changes might need to be made, but other than that, good progress!

**Review by narenrameshb — APPROVED** [review]
The changes look good and adhere to our discussions of steps forward in group meetings. As Enza mentioned in her review, some CA changes might need to be made, but other than that, good progress!

**Pull request merged**